### ISSUE TITLE
Add possibility to choose another calendar

### ISSUE THREAD

**Issue by Franzl123** [saschpe/BirthdayCalendar#7]
Hi,
Can u add the possibility to choose an other calendar for saving the data (eg. an caldav one)?

**Comment by saschpe** [comment]
Mhm, that would be a lot of work. If you just want a one-time export there are other apps which do just that. You could export the birthday calendar and be done. Continuously syncing to another caldav is different though.

**Comment by Franzl123** [comment]
No, i don't mean to implement an sync engine. In my case, i use davdroid for that (or even normally there is the Google one).

**Comment by saschpe** [comment]
In theory yes but I would have to redo some other parts as well. At the moment, the birthday calendar is regularly recreated (i.e. based on what changed in your contacts). I would have to use a more fine-grained method, i.e. only update those entries that actually changed. This way, the app wouldn't harm the other calendar entries...
